end2end/connectors/nakadi.py

```diff
@@ -255,7 +255,7 @@ def _on_event_pushed(r):
                 'metadata': {
                     'eid': str(uuid.uuid4()),
                     'event_type': self.topic,
-                    'occurred_at': datetime.now().isoformat()
+                    'occurred_at': datetime.now().replace(microsecond=0).isoformat()
```

Comment:
@antban what changes in the resulting string? As far as I know, it should accept microseconds.